tests/collections/tts/inference/test_spectrogram_synthesizers.py

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+import pytest
+import torch
+
+from nemo.collections.tts.inference.spectrogram_synthesizers import FastPitchSpectrogramSynthesizer
+from nemo.collections.tts.models import FastPitchModel
+
+
+class TestSpectrogramSynthesizers(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        super(TestSpectrogramSynthesizers, cls).setUpClass()
+
+        fastpitch_model = FastPitchModel.from_pretrained("tts_en_fastpitch_multispeaker").eval().to("cpu")
```

Comment:
it is fine to only download one ngc model and run evaluation here as a unit test, but I would suggest decorating it with a marker `nightly` and letting QA run the test. Our CI tried not to download NGC models for inference.

Reply:
see examples: https://github.com/NVIDIA/NeMo/tree/main/tests/collections/tts/models

Reply:
besides, be aware of the instance's life cycle for downloading the NGC model. Try not to duplicate downloading the NGC. `@fixture` decor could help.

Reply:
Added `@pytest.mark.nightly` and `@fixture` for all tests that download pretrained models.